content/docs/manuals/networking-internals/proxy-conversion.md

```diff
@@ -0,0 +1,256 @@
+
+```yaml
+{
+  "sla_version" : "v2.0",
+  "customerID" : "Admin",
+  "applications" : [
+    {
+      "applicationID" : "",
+      "application_name" : "X",
+      "application_namespace" : "default",
+      "application_desc" : "X application",
+      "microservices" : [
+        {
+          "microserviceID": "",
+          "microservice_name": "X1",
+          "microservice_namespace": "default",
+          "virtualization": "container",
+          "code": "docker.io/X/X1",
+          "addresses": {
+            "rr_ip": "10.30.0.1",
+            "rr_ip_v6": "fdff:2000::1"
+          },
+        },
+        {
+          "microserviceID": "",
+          "microservice_name": "X3",
+          "microservice_namespace": "default",
+          "virtualization": "container",
+          "code": "docker.io/X/X3",
+          "addresses": {
+            "rr_ip": "10.30.1.30",
+            "rr_ip_v6": "fdff:2000::30",
+          	},
+        }
+      ]
+    }
+  ]
+}
+```
+
+Therefore, we register into the platform two services, `X.default.X1.default` and `X.default.X3.default`.
+At deployment time, we request 2 instances of X1 (`X.default.X1.default.0` and `X.default.X1.default.1`)
+and one instance of X3 (`X.default.X3.default.0`). The scheduling decision places the instances as shown in
+the picture.
+From the deployment descriptor, we asked the platform to provision the Service IP `10.30.1.30` to X3 with Round
+Robin policy. Therefore, X1 will use this address to perform load-balanced requests toward X3.
+
+{{< /details >}}
+
+---
+
+Click on different steps to see what is happening behind-the-scenes.
+
+{{< tabs-icon "Requests" >}}
+{{< tab-icon "Step 1 - GET" "_step1.png">}}
+
+
+#### http://10.30.1.30:30443/api/hello
+
+X1 performs a GET request using the Service IP `10.30.1.30`. The default gateway for the `10.0.0.0/8` subnetwork
+is the ProxyTUN component of the Network Manager. The request will be directed there.
+From an L4 perspective, the packet will look somewhat like this:
+
+```
+from ip: 10.19.1.3
+to ip: 10.30.1.30
+from port: 34278
+to port: 30443
+payload: ....
+```
+
+The `from ip`, is the Virtual Layer IP, the **Namespace IP** of the container. This Namespace IP is assigned to the
+VETH device used to connect the container namespace to the virtual bridge in the system namespace.
+{{< /tab-icon >}}
+
+{{< tab-icon "Step 2 - Cache Miss" "_step2.png">}}
+
+#### Cache Miss
+
+When receiving the request packet, the proxy does not yet have an active conversion entry in its cache. This results
+in a cache miss. With a cache miss, the proxy tunnel fetches the information required for the conversion to the Environment
+Manager. This component keeps track of the services deployed internally in the worker node, as well as the relevant
+services deployed on other worker nodes.
+
+This is an example of the Conversion Table maintained by the Environment Manager at this moment.
+
+
+##### Node Service 1 Table before X3 table query
```

Comment:
Table exceeds content boundaries

<img width="1168" alt="Screenshot 2024-12-18 at 15 40 22" src="https://github.com/user-attachments/assets/a850a1db-4032-456b-b270-fe2095facf84" />

Reply:
This is a CSS problem, as the table in such a case should become scrollable.

Reply:
Solved it by rotating the table.